### PR TITLE
Add editor menu to open the project data "user://" location.

### PR DESCRIPTION
**Pull request by fire**
Add a way to open the project data folder. "user://"

image

The menu option is called "Open User Data/User Folder"

Edited:

Now "Open User Data Folder".

### CODE REVIEW

File: editor/editor_node.cpp [hunk 1]
```diff
@@ -2458,6 +2458,10 @@ void EditorNode::_menu_option_confirm(int p_option, bool p_confirmed) {
 
 			settings_config_dialog->popup_edit_settings();
 		} break;
+		case SETTINGS_USER_DATA_FOLDER: {
+
+			OS::get_singleton()->shell_open(String("file://") +	ProjectSettings::get_singleton()->globalize_path("user://"));
```
Comment (akien-mga) [inline editor/editor_node.cpp]: There's actually `OS::get_singleton()->get_user_data_dir()` to get this global path.

**Comment by fire** [comment]
image

File: editor/editor_node.cpp [hunk 2]
```diff
@@ -5944,6 +5948,7 @@ EditorNode::EditorNode() {
 #endif
 	p->add_separator();
 
+	p->add_item(TTR("Open User Data/User Folder"), SETTINGS_USER_DATA_FOLDER);
```
Comment (akien-mga) [inline editor/editor_node.cpp]: I guess "Open Use Data Folder" would be better?

**Review by akien-mga — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

**Comment by mrcdk** [comment]
isn't this the same as `Project/Open Project Data Folder`?

**Comment by akien-mga** [comment]
Indeed... Sounds like there is a discoverability issue for that option :)

I'll revert this PR then as it duplicates an existing feature.

**Pull request merged**